portal-api/.eslintrc.js

```diff
@@ -56,19 +57,22 @@ module.exports = {
         'comma-dangle': 'off',
         'lines-between-class-members': ['error', 'always', { exceptAfterSingleLine: true }],
         'no-loop-func': 'off',
-        'no-unused-vars': ['error'],
-        'object-curly-newline': ['error', {
-          consistent: true,
-        }],
+        'no-unused-vars': ['error', { argsIgnorePattern: '^draft', ignoreRestSiblings: true }],
```

Comment:
Allows for deep copying using immer,
`ignoreRestSiblings` to allow us to destructure/spread to split objects up to remaining items